roles/node_prep/tasks/45_networking_facts.yml

```
@@ -37,7 +37,7 @@
     ip: "{{ ansible_facts[provisioning_bridge]['ipv4']['address'] }}/{{ ansible_facts[provisioning_bridge]['ipv4']['netmask'] }}"
   when: (not enable_virtualmedia) and
         ((ipv4_provisioning|bool) or (not ipv6_enabled|bool) or
-        ((release_version.split('.')[0]|int == 4) and (release_version.split('.')[1]|int == 3)))
+        (release_version is ansible.builtin.version('4.3', '=='))
```

An another parenthesis to close
```suggestion
        (release_version is ansible.builtin.version('4.3', '==')))
```

---

True, changed